files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,14 @@
+
+module WriteChefError
+  class WriteChefError < Chef::Handler
+    def report
+      if run_status.failed?
```

Comment:
where is run_status coming from? if it's from Chef::Handler let's add this info in the description of the method/class, to be clear which are the inputs and the outputs of this method we're defining.